### ISSUE TITLE
At each epoch, loss of first batch is very different from subsequent batches 

### ISSUE THREAD

**Issue by sakina678** [fatchord/WaveRNN#36]
Hi @fatchord , thank you so much for sharing your great work!

I'm trying to train your alternate model. What I've noticed is that, at each epoch, the loss of the first batch is very different (usually much smaller) than the loss of subsequent batches. Is this normal? Why is this the case? Does that mean the model after the first batch is significantly better since it has a small loss? 

Here is an example:
<img width="346" alt="wavernn_loss_example" src="https://user-images.githubusercontent.com/44278144/51001725-bfd53c80-14e5-11e9-9a63-dc647aa087dc.png">

**Comment by fatchord** [comment]
@sakina678 It could be just coincidence in that the first batch has a lot of 'easy' samples to predict like silence. It should be loading the data randomly as Dataloader() has shuffle set to True in the notebook.
